parallel/spmv/parallel_row.jl

```diff
@@ -0,0 +1,19 @@
+using Finch
+using BenchmarkTools
+
+
+function parallel_row_mul(y, A, x)
+        _y = Tensor(Dense(Element(0.0)), y)
+        _A = Tensor(Dense(SparseList(Element(0.0))), A)
+        _x = Tensor(Dense(Element(0.0)), x)
+        time = @belapsed begin
+                (_y, _A, _x) = $(_y, _A, _x)
+                @finch mode = :fast begin
+                        _y .= 0
+                        for j = _, i = parallel(_)
```

Comment:
If you swap the order of i and j, and transpose the matrix outside of the kernel, this will be a much faster program.